src/VirtoCommerce.NotificationsModule.Web/Scripts/blades/notification-templates-list.js

```diff
@@ -1,155 +1,178 @@
+                                            return (element.languageCode === selection.languageCode);
+                                        });
+
+                                        if (index > -1) {
+                                            blade.currentEntity.templates.splice(index, 1);
+                                        }
                                     });
-        
-                                    if (index > -1) {
-                                        blade.currentEntity.templates.splice(index, 1);
-                                    }
                                 });
-                            });
+                            }
                         }
-                    }
-                };
-                dialogService.showConfirmationDialog(dialog);
+                    };
+                    dialogService.showConfirmationDialog(dialog);
+                }
             }
-            
-        }
-
-	    blade.toolbarCommands = [
-			    {
-				    name: "platform.commands.add", icon: 'fa fa-plus',
-				    executeMethod: createTemplate,
-				    canExecuteMethod: function () { return true; },
-				    permission: 'notifications:template:create'
+
+            blade.toolbarCommands = [{
+                name: "platform.commands.add",
+                icon: 'fa fa-plus',
+                executeMethod: createTemplate,
+                canExecuteMethod: function () { return true; },
+                permission: 'notifications:template:create'
+            }, {
+                name: "platform.commands.delete",
+                icon: 'fa fa-trash-o',
+                executeMethod: function () { deleteList($scope.gridApi.selection.getSelectedRows()); },
+                canExecuteMethod: function () {
+                    return $scope.gridApi && _.any($scope.gridApi.selection.getSelectedRows());
                 },
-                {
-                    name: "platform.commands.delete", icon: 'fa fa-trash-o',
-                    executeMethod: function () { deleteList($scope.gridApi.selection.getSelectedRows()); },
-                    canExecuteMethod: function () {
-                        return $scope.gridApi && _.any($scope.gridApi.selection.getSelectedRows());
-                    },
-                    permission: 'notifications:template:delete'
-                }
-	    ];
-
-        // ui-grid
-        $scope.setGridOptions = function (gridOptions) {
-          uiGridHelper.initialize($scope, gridOptions, function (gridApi) {
-              //update gridApi for current grid
-              $scope.gridApi = gridApi;
-              uiGridHelper.bindRefreshOnSortChanged($scope);
-          });
-          bladeUtils.initializePagination($scope);
-        };
-
-        blade.initialize();
-}]);
+                permission: 'notifications:template:delete'
+            }];
+
+            // ui-grid
+            $scope.setGridOptions = function () {
```

Comment:
Templates are defined in the html template, why we should move their usage and visual settings from there? I do not see the logic here that should be defined in js.